tests/test_apache-httpd.py

```diff
@@ -22,3 +23,11 @@ def test_apache_services(File, Service, Socket, AnsibleDefaults):
     assert Service("apache2").is_enabled
     assert Service("apache2").is_running
     assert Socket("tcp://0.0.0.0:" + str(http_port)).is_listening
+
+
+def test_apache_virtualhost():
+    virtualhost_test_text = 'idealista is awesome'
+    virtualhost_request = requests.get('http://apache.vm/example1/')
```

Comment:
I want to make this dependant on a variable, but not sure how to do it with Testinfra

Reply:
I think that it is ok. The test is couple to the  environment setup which is easy to maintain and understand